tests/appsec/iast/sink/test_command_injection.py

```diff
@@ -56,15 +56,17 @@ def test_secure(self):
     def setup_telemetry_metric_instrumented_sink(self):
         self.sink_fixture.setup_telemetry_metric_instrumented_sink()
 
-    @released(dotnet="?", golang="?", java="1.13.0", nodejs="?", php_appsec="?", python="?", ruby="?")
     @missing_feature(context.weblog_variant == "spring-boot-3-native", reason="GraalVM. Tracing support only")
+    @missing_feature(context.library < "java@1.13.0", reason="Not implemented yet")
+    @missing_feature(library="nodejs", reason="Not implemented yet")
```

Comment:
Don't we have to add the `missing_feature` macro for the other languages as well?

Reply:
@GustavoCaso Not if they are still not `@released`. No need to be exhaustive here for languages that have not implemented the feature described by the class at all.